src/mon/MonCommands.h

```diff
@@ -511,9 +511,6 @@ COMMAND("osd crush dump", \
 COMMAND("osd setcrushmap name=prior_version,type=CephInt,req=false", \
 	"set crush map from input file",	\
 	"osd", "rw", "cli,rest")
-COMMAND("osd crush set name=prior_version,type=CephInt,req=false", \
```

Comment:
my concern is that this change will break user's script. probably we could mark it deprecated, and remove it in next release? @liewegas what do you think?